covalent/_shared_files/config.py

```diff
@@ -24,12 +24,53 @@
 from functools import reduce
 from operator import getitem
 from pathlib import Path
-from typing import Any, Dict, List, Optional, Union
+from typing import Any, Dict, List, MutableMapping, Optional, Union
 
 import toml
+from dotenv import dotenv_values, find_dotenv, set_key
 
 """Configuration manager."""
 
+CONFIG_FILE_NAME = ".env"
+PROJECT_ROOT = os.path.dirname(os.path.abspath(__file__)) + "/../.."
```

Comment:
if the package is pip installed, this will point to `site-packages`... is that what we want?

Reply:
Given the config file is not intended to be edited directly by the user I think this suffices, I cannot think of other negative side effects of this as of now however.